clients/api-association/index.ts

```diff
@@ -1,31 +1,38 @@
-import { HttpNotFound } from "#clients/exceptions";
+import { HttpUnauthorizedError } from "#clients/exceptions";
 import routes from "#clients/routes";
 import type { IDataAssociation } from "#models/association/types";
 import constants from "#models/constants";
 import { formatAdresse, type IdRna, type Siren } from "#utils/helpers";
-import { clientAPIProxy } from "../client";
+import { httpGet } from "#utils/network";
 import type { IAssociationResponse } from "./types";
 
 /**
- * Association through the API proxy
+ * Wrapper client to call API Association
  *
  * Takes either a RNA or Siren, but Siren seems to work much better
- * @param idRna
+ * @param rnaOrSiren
+ * @param siretSiege
+ * @returns IDataAssociation
  */
-export const clientAssociation = async (
+export async function clientAPIAssociation(
```

Comment:
Why the use of function instead of const ?

Reply:
I used the code of another api-client as a start, and it was using `function` (api-entreprise/client)